website/install.rst

```diff
@@ -12,13 +12,13 @@ Install
              
 The Imath library is available for download and installation in
 binary form via package managers on many Linux distributions. See
-`https://pkgs.org/download/Imath
-<https://pkgs.org/download/Imath>`_ for a complete list. The common
-ones that generally provide current releases include:
+`Imath's page on Repology
```

Comment:
Thank you for the fixes! You might have seen that we recently added a similar reference to openexr, including an inline badge from repology listing the current versions. I was going to make a similar addition here but you've done it already, thanks! 

Could I ask you to mirror that here, with some slight modifications? I like the badge/table better than the list of Fedora, Gentoo, and Ubuntu, and the comment about out-of-date distributions applies to OpenEXR than to Imath, although it does warrant a reference to `ilmbase`. And homebrew is up to date now, too, so that comment should be struck.

```suggestion
Refer to the current version of Imath on various major Linux distros at
`repology.org <https://repology.org/project/imath/versions>`_:

.. image:: https://repology.org/badge/vertical-allrepos/imath.svg?exclude_unsupported=1&columns=4&header=Imath%20Packaging%20Status&minversion=3.0
   :target: https://repology.org/project/imath/versions

Older versions of Imath were distributed as a component of OpenEXR
called ``ilmbase``. We do not recommend using these outdated
versions.

To install via ``yum`` on RHEL/CentOS:

.. code-block::

    % sudo yum makecache
    % sudo yum install imath

To install via ``apt-get`` on Ubuntu:

.. code-block::

    % sudo apt-get update
    % sudo apt-get install imath


macOS
-----

On macOS, install via `Homebrew <https://formulae.brew.sh/formula/imath>`_:

.. code-block::

   % brew install imath

Alternatively, you can install on macOS via `MacPorts
<https://ports.macports.org/port/imath>`_:

.. code-block::

   % port install imath

Windows
-------

Install via `vcpkg <https://vcpkg.io/en/packages>`_:

.. code-block::

   % .\vcpkg install imath

```

Reply:
_Hello @cary-ilm,_

Thank you for the suggestion! 🙇🏼
I've applied it with minor adjustments.

> You might have seen that we recently added a similar reference to openexr, including an inline badge from repology listing the current versions.

No, I haven't 😄
I just noticed that the link to the Gentoo package is broken while I was working on gentoo/gentoo#35188. So I assumed it's worth checking other links. And I also decided to replace pkgs.org with Repology since it has a much broader coverage.